js/app/modules/card/list.js

```diff
@@ -30,11 +30,20 @@ const List = new Module.Class({
             'Show synopsis', 'Whether to show the synopsis label',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
             true),
+        /**
+         * Property: show-context
+         * Whether to show the context label.
+         */
+        'show-context': GObject.ParamSpec.boolean('show-context',
+            'Show context', 'Whether to show the context label',
+            GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
```

Comment:
At least for now, we make all properties construct-only if you can set them in the YAML